docs/API.md

```diff
@@ -553,17 +553,17 @@ An object with the following properties:
 * `"domain"`: A *domain name*, required. The zone to test.
 * `"ipv6"`: A boolean, optional. (default `false`). Used to configure the test and enable IPv4 tests.
 * `"ipv4"`: A boolean, optional. (default `false`). Used to configure the test and enable IPv6 tests.
-* `"nameservers"`: A list of *name server* objects, optional. Used to perform un-delegated test.
-* `"ds_info"`: A list of *DS info* objects, optional. Used to perform un-delegated test.
+* `"nameservers"`: A list of *name server* objects, optional. (default: `undef`). Used to perform un-delegated test.
```

Comment:
Well, in this repo (zonemaster-backend), we don't create an empty array when a ds_info/nameservers are not provided.
We save the request without modification about this two variables.

Reply:
Well, we don't update `nameservers` with `undef` before storing it in the database either.
The reason we specify default values here is so that it's clear how the method behaves when the argument is omitted. In this case omitting  `nameservers` results in behavior identical to what you get when calling it with `"nameservers": []`.

Reply:
done